src/Lykke.Service.PaySign.AzureRepositories/CallbackDataEntity.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using Common;
+using Lykke.AzureStorage.Tables;
+using Lykke.Service.PaySign.Core.Domain;
+
+namespace Lykke.Service.PaySign.AzureRepositories
+{
+    public class CallbackDataEntity : AzureTableEntity, ICallbackData
+    {
+        public static class ByDefault
+        {
+            public static string GeneratePartitionKey()
+            {
+                return "Default";
+            }
+
+            public static string GenerateRowKey()
+            {
+                return DateTime.UtcNow.ToIsoDateTime();
```

Comment:
DateTime.UtcNow.ToIsoDateTime doesn't have to be unique. You can have collisions